tasks/pull_date_format.js

```diff
@@ -0,0 +1,169 @@
+var path = require('path');
+var fs = require('fs');
+var constants = require('./util/constants');
+var common = require('./util/common');
+
+var args = process.argv.slice(2);
+var argLocale = args[0];
+
+var pathToEn = path.join(constants.pathToLib, 'locale-en.js');
+var pathToWCRegions = path.join(__dirname, '../node_modules/world-calendars/dist/regional');
```

Comment:
`path.join(require.resolve('world-calendars'), 'dist', 'regional')`

would be a bit more robust

Reply:
Neat, I didn't know about `require.resolve` - took a little fiddling but -> https://github.com/plotly/plotly.js/pull/2207/commits/d0e01584136a68d4373b7896ccbe758c9adbb527